sass/index.scss

```
@@ -0,0 +1,97 @@
+
+
+// Vars
+$logo-height: 100px;
+
+// Everything else
+
+html {
+}
+
+html, body {
+    overflow: hidden;
+    height: 100%;
+    width: 100%;
+}
+
+body {
+    background-color: #c0c0c0;
+    transform: scale(1.5);
+    transform-origin: center;
+}
+
+p {
+    // Reuse font from the library for p elmeents
+    font-family: "Pixelated MS Sans Serif", Arial;
+}
+
+footer {
+    position: absolute;
+    bottom: 18%; // From webapge scale
+    text-align: center;
+    width: 100%;
+}
+
+
+// Section specific styles
+
+#main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    gap: 30px;
+    margin-top: -$logo-height + 30px;
+}
+
+#logo {
+    width: $logo-height;
+    height: $logo-height;
+}
+
+.window {
+    width: 300px;
+    margin: 0 auto;
+}
+
+.field-row-stacked {
+
+    input {
+        width: 150px;
+    }
+
+    button {
+        margin: 0;
+        margin-left: 10px;
+        height: 21px; // Match the text box, hardcoded in the library :(
+        min-height: 0px;
+    }
+}
+
+#buttons {
+    width: 100%;
+    justify-content: center;
+    display: flex;
+    gap: 30px;
+}
+
+.text-break {
+    display: inline-block;
+}
+
+
```

slight nit: maybe add a section header comment here?
```suggestion

// Media queries

```